packages/big-design/src/components/Select/types.ts

```diff
@@ -18,10 +19,10 @@ export interface SelectProps<T> extends Omit<React.HTMLAttributes<HTMLUListEleme
   positionFixed?: boolean;
   required?: boolean;
   value?: T | T[];
-  onItemChange(value: T | T[], option: Option<T> | Array<Option<T>>): void;
+  onItemChange(value: (T | undefined) | T[], option: (Option<T> | undefined) | Array<Option<T>>): void;
 }
 
-interface BaseItem extends Omit<ListItemProps, 'children' | 'content' | 'value'> {
+interface BaseItem extends Omit<ListItemProps, 'children' | 'content' | 'isHighlighted' | 'isSelected' | 'value'> {
   content: string;
   icon?: React.ReactElement;
```

Comment:
Something discussed in Slack: The icon prop doesn't actually render an icon in `Selects`. Is this something you can fix in this PR?

Reply:
Will take a 👀